Core/src/org/sleuthkit/autopsy/communications/relationships/Bundle.properties-MERGED

```diff
@@ -35,20 +35,23 @@ MessageViewer_viewMessage_all=All
 MessageViewer_viewMessage_calllogs=Call Logs
 MessageViewer_viewMessage_selected=Selected
 MessageViewer_viewMessage_unthreaded=Unthreaded
-SummaryViewer.countsPanel.border.title=Counts
-SummaryViewer.contactsLabel.text=Contacts:
+SummaryViewer.countsPanel.border.title=Communications
+SummaryViewer.contactsLabel.text=Total Count:
 OutlineViewPanel.messageLabel.text=<Control Disabled>
 SummaryViewer.messagesDataLabel.text=messages
 SummaryViewer.callLogsDataLabel.text=callLogs
 SummaryViewer.contactsDataLabel.text=contacts
 SummaryViewer.messagesLabel.text=Messages:
 SummaryViewer.callLogsLabel.text=Call Logs:
+SummaryViewer_Account_Description=This account represents a device in the case.
+SummaryViewer_Account_Description_MuliSelect=Summary information is not available when multiple accounts are selected.
 SummaryViewer_CaseRefNameColumn_Title=Case Name
 SummaryViewer_CentralRepository_Message=<Enable Central Resposity to see Other Occurrences>
 SummaryViewer_Creation_Date_Title=Creation Date
+SummaryViewer_Device_Account_Description=This account was referenced by a device in the case.
+SummaryViewer_FileRef_Message=<Select one Accout to see File References>
```

Comment:
This message was just changed so the merged file needs to be updated as well.

Reply:
I don't understand why this didn't update.